reddit_about/pages.py

```diff
@@ -22,16 +22,7 @@ def __init__(self, content_id=None, title_msg=None, pagename=None, content=None,
         self.title_msg = title_msg
 
     def content(self):
-        about_buttons = [
-            NavButton(_('about reddit'), '/'),
-            NavButton(_('values'), '/values'),
-            NavButton(_('team'), '/team'),
-            NavButton(_('postcards'), '/postcards'),
-            NavButton(_('alien'), '/alien'),
-            #NavButton(_('guide'), '/guide')
-        ]
-        about_menu = NavMenu(about_buttons, type='tabmenu', base_path='/about/', css_class='about-menu')
-        return self.content_stack([AboutTitle(self.title_msg), about_menu, self._content])
+        return self.content_stack([AboutTitle(self.title_msg), self._content])
```

Comment:
Trying to look into whether `content_stack` needs `None` for where `about_menu` but the rabbit hole goes pretty deep - content_stack in BoringPage, to PaneStack where it's a pane, and so on.

So this works fine?